Tests/DependencyInjection/KnpDisqusExtensionTest.php

```diff
@@ -64,6 +64,17 @@ public function testDebugParameter()
         $this->assertEquals(1, $this->configuration->getParameter('knp_disqus.debug'));
     }
 
+    public function testCurlTimeoutParameter()
+    {
+        $this->createConfiguration('empty');
+
+        $this->assertEquals(Disqus::DEFAULT_TIMEOUT, $this->configuration->getParameter('knp_disqus.curl_timeout'));
+
+        $this->createConfiguration('full');
+
+        $this->assertEquals(8, $this->configuration->getParameter('knp_disqus.curl_timeout'));
```

Comment:
8 is actually equal to Disqus::DEFAULT_TIMEOUT, so it's not the best value to test with